lib/gradient/elixir_fmt.ex

```diff
@@ -143,13 +145,24 @@ defmodule Gradient.ElixirFmt do
     end
   end
 
-  def pp_expr(expression, _opts) do
-    IO.ANSI.blue() <> "#{inspect(expression)}" <> IO.ANSI.reset()
+  def pp_expr(expression, opts) do
+    fmt = Keyword.get(opts, :fmt_expr_fun, &ElixirExpr.pretty_print/1)
+
+    if Keyword.get(opts, :colors, true) do
+      IO.ANSI.blue() <> fmt.(expression) <> IO.ANSI.reset()
```

Comment:
Not necessarily in this PR, but we could give these colours abstract names, so that they're customisable by the user if need be. See [the IEx.configure section for an example of abstract names which are then mapped to colours](https://github.com/elixir-lang/elixir/blob/73d3f7dcb7c0e71f98dfd2a5c6e5cfb120b5c714/lib/iex/lib/iex.ex#L382).

Reply:
I've created an issue to track this - https://github.com/esl/gradient/issues/25